src/components/application_manager/include/application_manager/display_capabilities_builder.h

```diff
@@ -80,14 +80,27 @@ class DisplayCapabilitiesBuilder {
   /**
    * @brief ResetDisplayCapabilities resets stored notification
    */
-  void ResetDisplayCapabilities();
+  void ResetDisplayCapabilities(const WindowID window_id);
```

Comment:
@LitvinenkoIra please add a doxygen for a new param

Reply:
@AKalinich-Luxoft Done https://github.com/LuxoftSDL/sdl_core/pull/143/commits/f8f66c7c6ad7da8f2ac191942531918282b663df#diff-e3165ac166730ae95bbee625345e80edfac9afe6a2e4601fc5ee31400372491bR83